README.md

```diff
@@ -57,6 +57,8 @@ Web user interface for Batch Analysis
 
 ### Build and publish Docker image
 
+(You should not need to manually build or publish the image: it should happen automatically after your PR is merged to main).
```

Comment:
Not sure this is completely true, sometimes one wants to build an image from not yet merged code for testing.